chiron/chiron.js

```diff
@@ -1,18 +1,16 @@
-const got = require('got');
-const Logger = require('./js/Logger');
-const DatabaseModify = require('./js/databaseModify');
-const TrainingUrl = require('./models/trainingUrls');
-const ExpenseType = require('./models/expenseType');
-const Expense = require('expense-model');
 const _ = require('lodash');
+const got = require('got');
+const metascraper = require('metadata-scraper');
 const urlExists = require('url-exists');
+const Logger = require('Logger'); // from shared layer
+const DatabaseModify = require('databaseModify'); // from shared layer
+const TrainingUrl = require('trainingUrls'); // from shared layer
+const ExpenseType = require('expenseType'); // from shared layer
+const Expense = require('expense'); // from shared layer
 
```

Comment:
I think you said the Chiron function runs differently than other functions and you don't run it locally.  However, if you ever did, you would need the same ternary operator for these require statements, right?  For consistency, I think you should use the same ternary syntax here, even if you don't think you need it right now.